### DISCUSSION TITLE
Scrolling into really long message

### DISCUSSION THREAD

**Discussion by ghost** [tjtanjin/react-chatbotify#263] [Q&A]
Hello,

I'm having an issue where the chatbot always scroll into the `Option` section instead of the start of the Bot message. It would look okay if I have a short message, but when the message need to be really long then  I think this would be an UI/UX issue. Do you know what can I do to fix this issue? Thanks !

<img width="371" alt="Screenshot 2024-11-10 193556" src="https://github.com/user-attachments/assets/618c387b-3a64-4721-bb87-29e47ce2cc67">

**Comment by tjtanjin** [comment]
Hey @tphamcs, I'll find time to explore how best to address this. In the meantime, these are some considerations in mind:
- Current behavior mirrors existing messaging apps (new messages bring users to the bottom unless they are scrolling)
- If messages do not automatically scroll, users will need to manually scroll for every new message
- If messages automatically scroll, the beginning of long messages (such as yours) will be cut off

With these in mind, a good balance seems to be turning off automatic scrolling at the beginning until the user interacts with the chatbot. It is the likely solution I'll go with but we'll see.
